main.py

```diff
@@ -1,7 +1,50 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre_name in genres:
+        Genre.objects.create(
+            name=genre_name
+        )
+
+    for name, surname in actors:
+        Actor.objects.create(
+            first_name=name,
+            last_name=surname
+        )
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(
+        name="Action"
+    ).delete()
+    Actor.objects.filter(
+        first_name="Scarlett"
+    ).delete()
+    return Actor.objects.filter(
+        last_name="Smith"
+    ).order_by("first_name").all()
```

Comment:
No need to call `all` after `order_by`
The ordered queryset is already a representation of all the filtered objects in the desired order